package.json

```diff
@@ -20,7 +20,8 @@
     "test:passwords": "PASSWORD_STRESS_TEST=true jest generate.test.js",
     "test:unit": "jest",
     "test:integration": "npm run build && npm run copy-assets && playwright test",
-    "test:integration:ci": "xvfb-run --auto-servernum --server-args='-screen 0 1280x960x24' npm run test:integration -- --retries=3",
```

Comment:
I've removed these settings here because they tripped me up when trying to change the config file. Let me know if they were here for a reason I'm missing.